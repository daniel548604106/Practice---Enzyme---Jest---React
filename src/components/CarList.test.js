import { shallow } from 'enzyme';

import CardList from './CardList';

it('render CardList', () => {
  const mockRobots = [
    { id: 1, name: 'asadasd', username: 'Daniel', email: 'daniel@gmail.com' },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
