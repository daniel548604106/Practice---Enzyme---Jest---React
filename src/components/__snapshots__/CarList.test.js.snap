// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CardList 1`] = `
<div>
  <Card
    email="daniel@gmail.com"
    id={1}
    key="0"
    name="asadasd"
  />
</div>
`;
